pkg/cloud/rgraph/rnode/backendservice/type_trait.go

```diff
@@ -43,136 +43,30 @@ func (*typeTrait) FieldTraits(v meta.Version) *api.FieldTraits {
 	dt.OutputOnly(api.Path{}.Pointer().Field("SecurityPolicy"))
 	dt.OutputOnly(api.Path{}.Pointer().Field("SelfLink"))
 
-	dt.AllowZeroValue(api.Path{}.Pointer().Field("Iap"))
```

Comment:
I want to make sure I understand that removals in this file - we are removing the AllowZeroValue because now the FieldTypeOrdinary allows zero values?

Reply:
yes now FieldTypeOrdinary behaves as FieldTypeAllowZeroValue.
That mean that when API changed and a new field will be added UT tests will not be failing because we will trat new fields as AllowZeroValue.
As mentioned in comment before I will remove AllowZeroValue type when all resources will be migrated.

Reply:
sgtm. thanks for claryfying